jdbc/pom.xml

```diff
@@ -78,6 +78,29 @@
         <timestream.version>1.11.872</timestream.version>
     </properties>
 
+    <profiles>
```

Comment:
Need to ensure GitHub continues to sign by default. Usage of this profile might be missing in GitHub actions config yml file.

Reply:
Done, github actions has been modified